node/src/chain_spec/mod.rs

```diff
@@ -294,13 +294,13 @@ pub fn testnet_genesis(
         election: Some(CouncilElectionConfig {
             auto_start: true,
             election_parameters: ElectionParameters {
-                announcing_period: 2 * DAYS,
+                announcing_period: 4 * DAYS,
                 voting_period: 1 * DAYS,
                 revealing_period: 1 * DAYS,
-                council_size: 6,
-                candidacy_limit: 25,
+                council_size: 16,
+                candidacy_limit: 50,
                 min_council_stake: 1_000,
-                new_term_duration: 10 * DAYS,
+                new_term_duration: 1 * DAYS,
```

Comment:
Not sure why it is so low. No proposals will have time to be voted on and executed in time. Is that the intention?

Reply:
They can still vote throughout the announcing and voting period, so it shouldn't have any effect on that.
The purpose of this is to have very little downtime where you can't apply, nor vote for a CM.

An "actual" term will now last one week instead of two, and the incentives will be skewed heavily towards a CM's work instead of recurring rewards.